Gemfile

```diff
@@ -68,7 +74,7 @@ end
 
 group :test do
   # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
-  gem 'capybara'
-  gem 'selenium-webdriver'
-  gem 'webdrivers'
+  # gem 'capybara'
+  # gem 'selenium-webdriver'
+  # gem 'webdrivers'
```

Comment:
- [ ] As requested by the previous reviewer, please remove all unnecessary titles since they are empty  or add details to them (Prerequisites, run tests Deployment ....) 🚮